dbsync-central/release_stuff/scripts/arquive_older_debezium_bkps.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/sh
+# script for arquive the older backps of debezium files
+#
+
+# Set environment.
+HOME_DIR=/home/eip
+BKPS_HOME=$HOME_DIR/shared/bkps
+BKPS_ARQUIVED_HOME=$BKPS_HOME/arquived
```

Comment:
All back ups should be stored on the host and not inside the containers

Reply:
This is a volume dir located in host